x/staking/keeper/validator.go

```diff
@@ -32,15 +32,6 @@ func (k Keeper) mustGetValidator(ctx sdk.Context, addr sdk.ValAddress) types.Val
 	return validator
 }
 
-func (k Keeper) mustGetValidator(ctx sdk.Context, addr sdk.ValAddress) types.Validator {
```

Comment:
Why are we removing this?

Reply:
it was declared twice! (this used to be mustGetLiquidValidator and was renamed)